### ISSUE TITLE
jr git issue <repo> <title> | <desc>

### ISSUE THREAD

**Issue by fourjr** [verixx/selfbot.py#109]
have
GITHUBID
GITHUBPASS

in config vars and use some thing to make it happen

**Comment by verixx** [comment]
Good idea actually. @fourjr you should probably get started on it 👍. I'll catch up later.

**Comment by fourjr** [comment]
Using this as a testing issue, so ignore the "spam" here

**Comment by fourjr** [comment]
Added in latest few commits, with view issue/pr, submit issue, and add comment.

**Comment by fourjr** [comment]
Now, to include instructions on token in docs.

**Comment by nasirma1** [comment]
hi what do i have to put in github token ??
